application/features/Audio.py

```
@@ -166,18 +183,21 @@
 # if we are in debug mode, run the server in the second round
 if not app.debug or os.environ.get("WERKZEUG_RUN_MAIN") == "true":
     AUDIO_PORT = find_free_port()
-    print("AUDIO_PORT =", AUDIO_PORT)
+    # print("AUDIO_PORT =", AUDIO_PORT)
+    logger.debug("Audio: Audio port {}".format(AUDIO_PORT))
```

Ruff

Use f-string instead of `format` call

Convert to f-string

(UP032)